ecr-scripts/.terraform.lock.hcl

```diff
@@ -0,0 +1,45 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
I don't think we want to commit lock files to the module.

Reply:
will add a git ignore

Reply:
Still see the file here.